wgsl/index.bs

```diff
@@ -10218,6 +10222,19 @@ host-shared buffer, then:
 Note: Recall that [=i32=] uses twos-complement representation, so the sign bit
 is in bit position 31.
 
+When a value |V| of type [=u64=] is placed at byte offset |k| of a
```

Comment:
> &lt;dfn&gt;64-bit integer layout:&lt;/dfn&gt; Some features of the WebGPU API write 64-bit unsigned integer values into buffers.  When such a value |V| appears at byte offset |k| of a host-shared buffer, then:

...

> Note: WGSL does not have a [=concrete=] 64-bit integer type.


Then the WebGPU API should link to "64-bit integer layout"

Reply:
SGTM!